src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/transport/action/addpolicy/TransportAddPolicyAction.kt

```diff
@@ -73,7 +77,12 @@ class TransportAddPolicyAction @Inject constructor(
     }
 
     override fun doExecute(task: Task, request: AddPolicyRequest, listener: ActionListener<ISMStatusResponse>) {
-        AddPolicyHandler(client, listener, request).start()
+        val userStr = client.threadPool().threadContext.getTransient<String>(ConfigConstants.OPENDISTRO_SECURITY_USER_INFO_THREAD_CONTEXT)
+        user = resolveUser(User.parse(userStr))
```

Comment:
What does this parse as when security plugin isn't installed?

Reply:
I remember user saved is null when manual test

Reply:
userStr is null
parsed user is `parsed user: User[name=, backend_roles=[], roles=[], custom_attribute_names=[], user_requested_tenant=null]`

in the managed-index document:
```
"user": {
  "name": "",
  "backend_roles": [],
  "roles": [],
  "custom_attribute_names": [],
  "user_requested_tenant": null
}
```